src/components/todolist/todolist.tsx

```diff
@@ -245,47 +238,49 @@ const TodoList = (props: PropType) => {
     <div className="mt-10">
       <h1>Todo List</h1>
       <div className="d-flex align-items-start flex-row mt-8 pb-4 ">
-        <div className="mx-10">{captureForm()}</div>
+        <div className="mx-10">{<CaptureForm id={selectedID} />}</div>
         <div
+          className="d-flex"
           style={{
-            display: "flex",
             height: "400px",
-            borderWidth: "1px",
-            borderStyle: "solid",
-            borderColor: "gray",
+            border: "1px solid gray",
           }}
         ></div>
 
         <div
           style={{
             width: "30%",
-            overflowWrap: "break-word",
             marginLeft: 70,
           }}
         >
-          <DisplayTodos />
+          {props.todoItemsState.length > 0 ? (
+            <DisplayTodos />
+          ) : (
+            <Text>{"Your items will appear here"}</Text>
+          )}
+          {modalOpen && renderModal()}
         </div>
-        {actionPerformed && showToast()}
+        {props.toastState.toastVisible && renderToast()}
       </div>
     </div>
   );
 };
 
-function mapStateToProps(state: IState): IMapStateToProps {
+function mapStateToProps(state: IState) {
   return {
-    todoItemsState: state.todoItems,
+    todoItemsState: state.todos.todoItems,
+    toastState: state.toast,
   };
 }
 
-// need to import Dispatch from react, then pass the action type in Dispatch as type parameter
-function mapDispatchToProps(
-  dispatch: Dispatch<IDispatch>
-): IMapDispatchToProps {
+function mapDispatchToProps(dispatch: Dispatch<IDispatch>) {
   return {
     addTodoDispatch: (payload: string) => dispatch(addItem(payload)),
```

Comment:
Try it
```suggestion
    addTodoDispatch: addItem,
```

Reply:
Not giving any error, but also not dispatching the action